platform/mbed_power_mgmt.h

```diff
@@ -205,6 +205,32 @@ static inline void system_reset(void)
 {
     NVIC_SystemReset();
 }
+ 
+/** Provides the time spent in sleep mode, since system is up and running
+ *
+ *  @return  Time spent in sleep
+ */
+uint64_t mbed_time_sleep(void);
```

Comment:
why was `uint64_t` chosen as a unit to be returned here ? Don't we have time based user defined variables - `us_timestamp_t` ? Based on the implementation, I would expect that us timestamp would fit here? As it is, I am missing the units returned (its microseconds after looking at the implementation).

Reply:
us_timestamp_t is perfect fit, but its declaration is guarded by `DEVICE_LOWPOWERTIMER`, hence will get compilation errors for targets not supporting LP.

Reply:
@c1728p9 - Can i move `#include "hal/ticker_api.h"` outside `#if DEVICE_LOWPOWERTIMER` define?